src/hardware/mixer.cpp

```diff
@@ -931,16 +843,21 @@ void MIXER_Init(Section* sec) {
 	mixer.mastervol[0]=1.0f;
 	mixer.mastervol[1]=1.0f;
 
+	/* Calculate blocksize from requested latency to nearest power of 2 */
+	const auto clamped_latency = clamp(static_cast<int>(mixer.latency), 1, 100);
```

Comment:
The setup code where you've set the range between 1 and 100 will hard-guarantee the value is in range (if it's our of bounds, it warns the user and then sets the default). 
Because it's our own code, suggest simply asserting that the value is <= 100.